RwsCheck.py

```diff
@@ -496,10 +491,8 @@ def find_robots_txt(self, check_sets):
                                         "'noindex' or 'none' tag in its header"
                                         )
                 except Exception as inst:
-                    if exception_retries not in str(inst):
-                        if exception_timeout not in str(inst):
-                            self.error_list.append(
-                                f"Unexpected error for service site: {service_site}; Received error: {inst}")
+                        self.error_list.append(
```

Comment:
Looks like this is indented one level too far. (It'd be really nice if we had an autoformatter for the python in this repo.)